apps/android/boot.js

```diff
@@ -49,11 +49,15 @@
       },
       // {t:"musicstate", state:"play/pause",position,shuffle,repeat}
       "musicstate" : function() {
-        require("messages").pushMessage({t:"modify",id:"music",title:"Music",state:event.state});
+        require("messages").pushMessage({t:"modify",id:"music",title:"Music",state:event.state,position:event.position,src:"musicstate"});
       },
       // {t:"musicinfo", artist,album,track,dur,c(track count),n(track num}
       "musicinfo" : function() {
-        require("messages").pushMessage(Object.assign(event, {t:"modify",id:"music",title:"Music"}));
+        require("messages").pushMessage(Object.assign(event, {t:"modify",id:"music",title:"Music",src:"musicinfo"}));
+      },
+      // {t:"audio", l(level):{c:[current volume level], u:[upper volume level]}}
```

Comment:
Is this in Gadgetbridge yet? Please can you make sure it all gets documented in https://github.com/espruino/EspruinoDocs/blob/master/info/Gadgetbridge.md ?